src/shared/src/utils/instanceIdRegExp.ts

```diff
@@ -0,0 +1,23 @@
+const instanceIdRegExp =
+  /(\d{1,10}\/[\d,a-f]{8}-[\d,a-f]{4}-[1-5][\d,a-f]{3}-[89ab][\d,a-f]{3}-[\d,a-f]{12})/i;
```

Comment:
```suggestion
  /(\d{1,10}\/[\da-f]{8}-[\da-f]{4}-[1-5][\da-f]{3}-[89ab][\da-f]{3}-[\da-f]{12})/i;
```
I'm assuming we're matching a UUID here, and that it shouldn't be matching commas? This regex would also match the string `123456/7515,373-aed4-41f7-95b4-e5b5115c2edc` (notice the comma).

Also not quite sure where this regex came from, but I can't imagine it being important that some of these parts of the UUID only start with `[1-5]` and `[89ab]`? If not, it can be simplified further:

```suggestion
  /(\d{1,10}\/[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12})/i;
```